tests/examples/pacer/nef/s3/cob.json

```diff
@@ -0,0 +1,26 @@
+{
+  "appellate": false,
+  "contains_attachments": false,
+  "court_id": "cob",
+  "dockets": [
+    {
+      "case_name": "Delta, LLC",
+      "date_filed": null,
+      "docket_entries": [
+        {
+          "date_filed": "2023-09-25",
+          "description": "Motion to Dismiss Case For Other Reasons the debtor entity does not have a need for bankruptcy protection Filed by FilerFirstname FilerLastName on behalf of Delta, LLC. (Attachments: (1) Proposed/Unsigned Order) (FilerLastName, FilerFirstname) The following document(s) are associated with this transaction: Document description: Main Document Original filename: Dismissal.pdf Electronic document Stamp: [STAMP bkecfStamp_ID=985638001 [Date=9/25/2023] [FileNumber=48774698-0] [ad540b5d97ef1738274ab45e91c33cb34d46c4ea12d40070860aa383a9c33d8bade5e4ceb25f28af8647ab34d0ce7085a93200a15270c5a5e51353a7a79edba9]] Document description: Proposed/Unsigned Order Original filename: C:\\fakepath\\Ex A - Proposed Order.pdf Electronic document Stamp: [STAMP bkecfStamp_ID=985638001 [Date=9/25/2023] [FileNumber=48774698-1] [06f97e6a23f90c93e04b24659af0d156b12d3694b68f527325fee1a54f101f3e5cf9587ff565009042203876d040b7e0bf418eeb94635534be99f134d37f999d]] 23-14130-JGR Notice will be",
```

Comment:
I'm noticing that in this example, the description is not being parsed properly.

The description should be only: `Entry of Appearance and Request for Notice Filed by  Eric J. Langston   on behalf of    Delta, LLC...  (Langston, Eric)`

I also tested the other examples that you listed in: https://github.com/freelawproject/juriscraper/issues/729#issuecomment-1901443139 and the behaviour is similar.

Reply:
I have fixed this, please check again